electron-app/src/server/websites/pixelfed/pixelfed.service.ts

```diff
@@ -180,85 +195,69 @@ export class Pixelfed extends Website {
     }
 
     const instanceInfo: PixelfedInstanceInfo = this.getAccountInfo(data.part.accountId, INFO_KEY);
-    const chunkCount = instanceInfo
-      ? instanceInfo?.configuration?.statuses?.max_media_attachments
-      : 4;
+    const chunkCount = instanceInfo ? instanceInfo?.configuration?.statuses?.max_media_attachments : 4;
     const maxChars = instanceInfo ? instanceInfo?.configuration?.statuses?.max_characters : 500;
 
     const isSensitive = data.rating !== SubmissionRating.GENERAL;
     const { options } = data;
     const chunks = _.chunk(uploadedMedias, chunkCount);
     let lastId = undefined;
+    let statusOptions: any = {
+      sensitive: isSensitive,
+      visibility: options.visibility || 'public',
+      in_reply_to_id: lastId,  
+      spoiler_text: "",    
+    }  
+    let status = "";
+
     for (let i = 0; i < chunks.length; i++) {
-      let form = undefined;
       if (i === 0) {
-        form = {
-          status: `${options.useTitle && data.title ? `${data.title}\n` : ''}${
-            data.description
-          }`.substring(0, maxChars),
-          sensitive: isSensitive,
-          visibility: options.visibility || 'public',
-          media_ids: chunks[i].map((media) => media.id),
-        };
-      } else {
-        form = {
-          sensitive: isSensitive,
-          visibility: options.visibility || 'public',
-          media_ids: chunks[i].map((media) => media.id),
-          in_reply_to_id: lastId,
-        };
-      }
+        status = `${options.useTitle && data.title ? `${data.title}\n` : ''}${
+          data.description
+          }`.substring(0, maxChars);
+        statusOptions.media_ids = chunks[i].map((media) => media.id);
+     }
 
-      const tags = this.formatTags(data.tags);
+      this.logger.debug(`Number of tags set ${data.tags.length}`);
 
-      // Update the post content with the Tags if any are specified - for Pixelfed, we need to append
+      // Update the post content with the Tags if any are specified - for Pixelfed, we need to append 
       // these onto the post, *IF* there is character count available.
-      if (tags.length > 0) {
-        form.status += '\n\n';
+      if (data.tags.length > 0) {
+        status += "\n\n";
       }
 
-      tags.forEach((tag) => {
-        let remain = maxChars - form.status.length;
+      data.tags.forEach(tag => {
```

Comment:
And here.

I just don't want any tagging regressions. See formatTags in the code diff.

Reply:
Resolved